web/docs/connectors/uri.md

```diff
@@ -0,0 +1,42 @@
+# URI
```

Comment:
@mavam Do you think it makes more sense to document URIs as a connector, or to expand the synopsis of operators working with URIs like this?

```
from <uri>
from <file>
from <connector>
from <connector> read <format>
```

I am undecided between the two, I don't know what's better.

Reply:
I don't think it makes sense as separate connector. If we did, I'd expect something like `from uri <uri>`. Rather, I think we should lift it to the parent context, similar to what we did with modifiers.

In particular, we could put this into `/connectors` and `/formats` where we already write:

```
from <connector> [read <format>]
to <connector> [write <format>]
```

The only thing I'm still thinking about is the duplication of content here.

Reply:
I agree that `URI` should not be documented as a proper connector, and that we should not duplicate the path/URI/loader/saver description four times. Adding it to the synopses seems fine though, although it might be a bit inconsistent with how we use `<connector>`. But I would be okay with that (though I would probably use `<path>` instead of `<file>` to prevent confusion with `file`). `/connectors` seems like a good place to document this once, and we can link to that from the other operators. `/formats` could even stay as-is. There should not be much duplication then, right?

Reply:
I like that approach!

Reply:
I did some changes. Is it aligned with what you had in mind, now?

Reply:
That's more or less the opposite direction. The suggestion was to move the URI and path docs to `/connectors`, that is: `web/docs/connectors.md`. The goal is to only have "actual" connectors (which URI and path are not) in `web/docs/connectors/*.md`.

Reply:
Ah, got it. I got the directory and the file confused. Let's try again!

Reply:
Yes, that's better.